go/gorouter/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12.5
+
+WORKDIR /go/src/app
+COPY main.go go.mod ./
```

Comment:
missing `go.sum` file?

Reply:
added go.sum

Reply:
@appleboy Is `go.sum` really required for a benchmark use-case ?